p2p/src/main/kotlin/maru/p2p/MaruPeer.kt

```diff
@@ -28,6 +33,9 @@ import tech.pegasys.teku.networking.p2p.rpc.RpcRequestHandler
 import tech.pegasys.teku.networking.p2p.rpc.RpcResponseHandler
 import tech.pegasys.teku.networking.p2p.rpc.RpcStreamController
 
+private const val STATUS_RENEWAL_SECONDS = 300L // 5 minutes
```

Comment:
We will need this configurable. For the testing we need much faster feedback loop